week4/MealsharingNET/Models/Meal.cs

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
You are missing a line here defininig the namespace of this class, should be something like namespace MealsharingNET.Models